SU-SOE/soe_profile
# SOEOP-339: Created `site_embedder` role.

# READY FOR REVIEW

# Summary
- SOE has requested an additional role which has the same permissions as `site_manager` but with the addition of using the freeform embeddables field, normally reserved for `site_developer` and `administrator`.  This PR creates that role and has tests to verify.

# Needed By (Date)
- 1.5.0 codefreeze.

# Urgency
- 2 out of 10.

# Steps to Test

1. Pull this profile into a lelandd8 stack.
2. `drush cim` to get the configs in.
3. Create a user with the "Site Embedder" role.
4. Log in as that user and verify that the unstructured embed field works as expected.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOEOP-339

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
